.gitignore

```diff
@@ -1,2 +1,2 @@
-/vendor/
-/config.php
+/.cache/
+/venv/
```

Comment:
no need for a `__pycache__` folder or `*.pyc` files? (varies between versions)

Reply:
Added `__pycache__`.